legion/engine/application/input/inputsystem.hpp

```diff
@@ -539,12 +540,43 @@ namespace legion::application
             }
         }
 
+        template <class Event>
+        void pushCommand(float value,inputmap::modifier_keys mods, inputmap::method method)
+        {
+            auto& cq = m_axes_command_queues[Event::id];
+            cq.values.push_back(value);
+            cq.mods.push_back(mods);
+            cq.methods.push_back(method);
+        }
+
+        void raiseCommandQueues(float delta)
+        {
+
+            for(auto  [key,value] : m_axes_command_queues){
+                auto axis = std::make_unique<input_axis<std::nullptr_t>>();
+                axis->value_parts = value.values;
+                axis->mods_parts = value.mods;
+                axis->identifier_parts = value.methods;
+
+                axis->input_delta = delta;
+                axis->value = std::accumulate(value.values.begin(),value.values.end(),0.0f);
+
+                raiseEventUnsafe(std::move(axis),key);
```

Comment:
Are you sure you want to raise it unsafe?

Reply:
**Y̶̤̮͗͑̎̎̈̅̓͋̈͐̈̂̍͘Ë̶͖́͑̈́̑̀́̌̚Ş̸̫̬̰̯͕̦̟͈̬̻̣̟̣̽̾̀̍́̚͜**